test/packages/bad_custom_ilm_policy/docs/README.md

```diff
@@ -0,0 +1,84 @@
+<!-- Use this template language as a starting point, replacing {placeholder text} with details about the integration. -->
+<!-- Find more detailed documentation guidelines in https://github.com/elastic/integrations/blob/main/docs/documentation_guidelines.md -->
+
+# This package has a valid custom ILM policy.
```

Comment:
Should the title be updated to mention that it has an invalid ILM Policy ? 

```suggestion
# This package has an invalid reference to a custom ILM policy.
```

If so, there are also some references in the following paragraphs